--- acs-0.29/modelgen/Make2.g++.orig	Thu Feb 13 21:54:02 2003
+++ acs-0.29/modelgen/Make2.g++	Thu Feb 13 21:54:54 2003
@@ -5,7 +5,11 @@
 
 CCC = g++
 CCFLAGS = \
--DUNIX -O3 -DNDEBUG -I.. -fexceptions -Wall -W  
+-DUNIX $(OPT_FLAGS) -DNDEBUG -I.. -fno-exceptions \
+-Wall -W -Wno-sign-compare \
+-Wundef -Wpointer-arith -Wbad-function-cast -Wcast-qual \
+-Wwrite-strings -Wconversion \
+
 LIBS = -lm
 LDFLAGS =
 
--- acs-0.29/src/Make2.g++.orig	Thu Feb 13 21:58:02 2003
+++ acs-0.29/src/Make2.g++	Thu Feb 13 21:58:21 2003
@@ -5,7 +5,11 @@
 
 CCC = g++
 CCFLAGS = \
--DUNIX -O3 -DNDEBUG -I.. -fno-exceptions -Wall -W  
+-DUNIX $(OPT_FLAGS) -DNDEBUG -I.. -fno-exceptions \
+-Wall -W -Wno-sign-compare \
+-Wundef -Wpointer-arith -Wbad-function-cast -Wcast-qual \
+-Wwrite-strings -Wconversion \
+
 LIBS = -lm
 LDFLAGS =
 
--- acs-0.29/src/Make2.sun-CC.orig	Thu Feb 13 21:57:02 2003
+++ acs-0.29/src/Make2.sun-CC	Thu Feb 13 21:57:26 2003
@@ -4,7 +4,7 @@
 
 # Use g++ if you have it for a speed advantage of about 20%
 CCC = CC
-CCFLAGS = -DNDEBUG -DUNIX -O4
+CCFLAGS = -DNDEBUG -DUNIX $(OPT_FLAGS)
 LIBS = -lm
 LDFLAGS =
 
